aten/src/ATen/Dispatch.h

```diff
@@ -148,6 +148,20 @@ inline void deprecated_AT_DISPATCH_ALL_TYPES_AND_HALF_AND_COMPLEX() {}
     }                                                                        \
   }()
 
+#define AT_DISPATCH_FLOATING_TYPES_AND_LONG(TYPE, NAME, ...)                 \
```

Comment:
It might be nicer to add a `AT_DISPATCH_FLOATING_TYPES_AND(SCALARTYPE, TYPE, NAME, ...`
because there is already `AT_DISPATCH_FLOATING_TYPES_AND_HALF` and we might want a `AT_DISPATCH_FLOATING_TYPES_AND_HALF_AND_LONG` in the future...

Reply:
agreed.

Reply:
Fixed